scripts/core/EXP-LAUNCH-ATTRIBUTES.rst

```diff
@@ -0,0 +1,102 @@
+<%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+
+.. _experimental-launch-properties:
+
+================================================================================
+LAUNCH Properties
+================================================================================
+
+.. warning::
+
+    Experimental features:
+
+    *   May be replaced, updated, or removed at any time.
+    *   Do not require maintaining API/ABI stability of their own additions over
+        time.
+    *   Do not require conformance testing of their own additions.
+
+
+Terminology
+--------------------------------------------------------------------------------
+"Launch Properties" is used to indicate optional kernel launch properties that
+can be specified at the time of a kernel launch. Such properties can be used to
+enable hardware specific kernel launch features.
+
+Motivation
+--------------------------------------------------------------------------------
+Advances in hardware sometimes require new kernel properties. One example is
+distributed shared memory as used by Nvidia Hopper GPUs. Launching a kernel
+that supports distributed shared memory requires specifying a set of "cluster"
+dimensions, in units of work-groups, over which the shared memory is
+"distributed". Additionally some applications require specification of kernel
+properties at launch-time.
+
+This extension is a future-proof and portable solution that supports these two
+requirements. Instead of using a fixed set of kernel enqueue arguments, the
+approach is to introduce the `exp_launch_property_t` type that enables a more
+extendable API.
+
+Each `exp_launch_property_t` instance corresponds to a specific kernel launch
+property.
+Only one new function is introduced: `urEnqueueKernelLaunchCustomExp`.
+`urEnqueueKernelLaunchCustomExp` takes an array of `exp_launch_property_t` as
+an argument, and launches a kernel using these properties.
+`urEnqueueKernelLaunchCustomExp` corresponds closely to the CUDA Driver API
+`cuLaunchKernelEx`.
```

Comment:
Single backticks render as italics in reStructuredText. Also when using `${x}` for the ur prefixes those don't render well in inline monospace because the templates generate cross-links. 

```suggestion
This extension is a future-proof and portable solution that supports these two
requirements. Instead of using a fixed set of kernel enqueue arguments, the
approach is to introduce the ${x}_exp_launch_property_t type that enables a
more extendable API.

Each ${x}_exp_launch_property_t instance corresponds to a specific kernel
launch property.
Only one new function is introduced: ${x}EnqueueKernelLaunchCustomExp.
${x}EnqueueKernelLaunchCustomExp takes an array of ${x}_exp_launch_property_t
as an argument, and launches a kernel using these properties.
${x}EnqueueKernelLaunchCustomExp corresponds closely to the CUDA Driver API
``cuLaunchKernelEx``.
```

Reply:
Thanks looks good.